react/src/components/BAINotificationItem.tsx

```diff
@@ -18,7 +18,8 @@ const BAINotificationItem: React.FC<{
     notification: NotificationState,
   ) => void;
   showDate?: boolean;
-}> = ({ notification, onClickAction, showDate }) => {
+  destroyAll?: () => void | undefined;
```

Comment:
This `prop` is function type, it's better to use `onClickDestroyAll` name patter

Reply:
Resolved